demo/api/src/products/products.module.ts

```diff
@@ -46,7 +48,9 @@ import { ProductVariantResolver } from "./generated/product-variant.resolver";
         ManufacturerResolver,
         ManufacturerCountryResolver,
         ProductToTagResolver,
+        ProductImporterConsole,
+        ProductImporterService,
     ],
-    exports: [],
+    exports: [ProductImporterService],
```

Comment:
Why do you need to export the service?

Reply:
Ah now that the product importer doesn't have its own module anymore, we don't have to export it anymore

Reply:
```suggestion
    exports: [],
```

Then let's remove it.